examples/proxy/record-validation/cluster-ip/README.md

```diff
@@ -0,0 +1,69 @@
+# Streams for Apache Kafka Proxy Record Validation, exposed using Cluster-IP
+
+In this example, an instance of Apache Kafka is deployed using Streams for Apache Kafka.  Apicurio
+Registry is deployed to the cluster too.
+
+The Streams Proxy is deployed with configuration to perform record validation.  The configuration ensures that 
+any records sent to a topic called `people` adhere to a `person` schema.
+
+Finally, kafka command line (run on cluster) tools are used to send valid and invalid record to the `people` topic
+so that the effects of the validation can be observed.
+
+# Prerequisites
+
+* Administrative access to the OpenShift Cluster being used to evaluate Streams for Apache Kafka Proxy
+* Streams for Apache Kafka Operator (installed cluster wide)
+* Red Hat build of Apicurio Registry Operator (installed cluster wide)
+* OpenShift CLI (oc)
+* cURL
+
+# Deploying the Example
+
+1. Deploy the Example:
+   ```sh
+   oc apply -k cluster-ip
+   oc wait deployment/proxy --for=condition=Available=True --timeout=300s -n kafka-proxy
+   ```
+
+2. Create the example schema in the registry:
+
+   ```sh
+   REGISTRY_URL=http://$(oc get apicurioregistries.registry.apicur.io -n schema-registry registry --template='{{.status.info.host}}')
+   curl -X POST ${REGISTRY_URL}/apis/registry/v2/groups/default/artifacts -H "Content-Type: application/json; artifactType=JSON" -H "X-Registry-ArtifactId: Person" --data @schemas/person.schema.json
+   ```
+
+# Try out the example
+
+1. Create a topic `people` on the cluster, via the proxy:
+   ```sh
+   oc run -n kafka-proxy -qi create-topic --image=registry.redhat.io/amq-streams/kafka-37-rhel9:2.7.0 --rm=true --restart=Never -- bin/kafka-topics.sh --bootstrap-server proxy-service:9092 --create -topic people
+   ```
+2. Produce a record to the topic with a record value that matches the schema.
+   ```sh
+   cat record-examples/valid-person.json | oc run -n kafka-proxy -qi proxy-producer --image=registry.redhat.io/amq-streams/kafka-37-rhel9:2.7.0 --rm=true --restart=Never -- bin/kafka-console-producer.sh --bootstrap-server proxy-service:9092 --topic people --sync
```

Comment:
It'd been nice to illustrate an example where the producer is using the schema itself.    Unfortunately we can't do that with command line tools available to us.

* The kafka-console-producer shipped by Streams doesn't have the necessary SerDes in its classpath (perhaps it should?).
* kcat` and `kaf` have the ability to produce messages referencing schemas, but we don't have a Red Hat build of these tools to reference.    Also these tools talk Confluent schema identifiers.   That won't work unti https://github.com/Apicurio/apicurio-registry/issues/5086 is done.

Reply:
I think my comment is Streams wide, I've raised a JIRA https://issues.redhat.com/browse/ENTMQST-6324